release-notes/release-notes56.md

```diff
@@ -24,14 +24,16 @@ These release notes provide an overview of the changes for {{site.productname}}
 
 The following new features and enhancements were added for the {{site.productname}} 5.6 release.
 
-### New `image_file_types` option for customizing the recognized `smart_paste` image file extensions
+### New `image_file_types` option for customizing recognized image file extensions
 
-The new `image_file_types` option allows different image file extensions to be recognized by the `smart_paste` image URL to embedded image functionality.
+The new `image_file_types` option allows different image file extensions to be recognized by the `smart_paste` image URL to embedded image functionality, as well as image file uploads.
```

Comment:
This is a bit messy now, would love your input @tylerkelly13

Reply:
Try converting it to a unordered list